source/playbooks/NEWPLAYBOOK/test/newplaybook_stack.test.ts

```diff
@@ -20,26 +21,28 @@ function getTestStack(): cdk.Stack {
   return stack;
 }
 
-test('default stack', () => {
-  expect(SynthUtils.toCloudFormation(getTestStack())).toMatchSnapshot();
+test('admin stack', () => {
+  expect(Template.fromStack(getTestStack())).toMatchSnapshot();
 });
 
-function getMemberStack(): cdk.Stack {
-  const app = new cdk.App();
+function getMemberStack(): Stack {
+  const app = new App();
   const stack = new PlaybookMemberStack(app, 'memberStack', {
+    synthesizer: new DefaultStackSynthesizer({ generateBootstrapVersionRule: false }),
     description: 'test;',
     solutionId: 'SO0111',
     solutionVersion: 'v1.1.1',
     solutionDistBucket: 'sharrbukkit',
     ssmdocs: 'playbooks/NEWPLAYBOOK/ssmdocs',
-    remediations: [{ control: 'Example.3' }, { control: 'Example.5' }, { control: 'Example.1' }],
-    securityStandard: 'PCI',
-    securityStandardLongName: 'pci-dss',
+    remediations: [{ control: 'RDS.6' }],
+    securityStandard: 'NPB',
+    securityStandardLongName: 'newplaybook',
```

Comment:
I'm a little confused by this change, maybe I just don't understand the purpose of the test?

Reply:
This test was never running. These values control where it searches for remediation documents. The example was wrong before and didn't build properly. With these changes it is correct and builds.